views/sections/library.html

```
@@ -62,3 +61,88 @@ <h1 class="text-3xl font-bold text-slate-800 dark:text-white">Music Library</h1>
     </div>
   </div>
+
+  <!-- Mini player bar (shows when a track is clicked) -->
+  <div id="player-bar"
+       class="hidden fixed bottom-0 inset-x-0 sm:left-64 z-40 border-t border-white/40 dark:border-white/10 bg-white/20 dark:bg-neutral-900/25 backdrop-blur-2xl shadow-[0_-6px_30px_rgba(142,197,255,0.18)]">
+    <div class="w-full px-3 sm:px-6 py-2.5">
+      <div class="grid grid-cols-[1fr_auto_1fr] items-center gap-3 sm:gap-4">
+        <!-- Col 1: artwork + title -->
+        <div class="flex items-center gap-3 min-w-0">
+          <div class="relative flex-shrink-0 w-12 h-12 sm:w-14 sm:h-14 rounded-xl overflow-hidden bg-gradient-to-br from-sky-400 to-indigo-600 shadow-lg">
+            <img id="player-art" src="" alt="" class="w-full h-full object-cover" _="on error set my style.display to 'none'">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)